README.md

```diff
@@ -63,7 +63,8 @@ jake runtests         # Run tests using the built compiler and test infrastructu
                       # Use host=<hostName> or tests=<testPath>. 
 jake runtests-browser # Runs the tests using the built run.js file. Syntax is jake runtests. Optional
                         parameters 'host=', 'tests=[regex], reporter=[list|spec|json|<more>]'.
-jake baseline-accept  # This replaces the baseline test results with the results obtained from jake runtests. 
+jake baseline-accept  # This replaces the baseline test results with the results obtained from jake runtests.
+jake lint             # Runs tslint over the compiler

```

Comment:
Right now 'jake lint' runs over the compiler and harness code but not the language service, presumably in the future it'll include the language service and maybe other stuff. So maybe just say 'Runs tslint over the TypeScript sourcecode' or something a little more agnostic.